### PR TITLE
ISO Code update for Odisha state (India)

### PR DESCRIPTION
**Pull request by sorensd**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

ISO Code change for Odisha state (India)

ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add products to the cart and proceed to checkout.
2. On the Country dropdown select `India` and on the State select `Odisha`.
3. Check the code for the select is `OD` instead of `OR`.

Screenshot 2024-12-10 at 09 50 08

### Changelog entry
Updated Odisha ISO Code - 23 November 2023

-   [X] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: packages/js/onboarding/changelog/53341-trunk [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+ISO Code update for Odisha state (India) https://en.wikipedia.org/wiki/ISO_3166-2:IN (23 November 2023)
```
Comment (albarin) [inline packages/js/onboarding/changelog/53341-trunk:4]: @sorensd would you mind removing this changelog file? 🙏 somehow it got pushed but the correct one is the `plugins/woocommerce/changelog/53341-trunk`

**Review by albarin — CHANGES_REQUESTED** [review]

**Review by sorensd — COMMENTED** [review]
This is done.

**Review by sorensd — COMMENTED** [review]
@albarin I am not very sure how the packages/js/onboarding/changelog/53341-trunk is getting created? I am stuck on a loop

**Comment by albarin** [comment]
@sorensd Sorry about this, I just realized it's probably created automatically because there are changes to this file `packages/js/onboarding/src/utils/countries/test/utils/country-options.ts`, I think it's okay to leave it 🙏

**Review by albarin — APPROVED** [review]

**Pull request merged**